### PR TITLE
12113 scaffold wizard

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/12113)

This PR is to add the code to build the pages of the wizard for chapter 31. This also adds in the questions as we know them at this point in time (this is still going through design reviews). Unit tests will be added in a separate PR

## Testing done
Unit tests will be added in a separate branch

## Acceptance criteria
- [x] The CH31 wizard is scaffolded and ready for next steps
- [x] A PR has been submitted for merging

### CODE REVIEW

File: src/applications/vre/28-1900/wizard/pages/service-member/03-noIDES.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ErrorableRadioButtons from '@department-of-veterans-affairs/formation-react/ErrorableRadioButtons';
+import { serviceMemberPathPageNames } from '../pageList';
+
+const noIDES = ({ setPageState, state = {} }) => (
+  <div className="feature">
+    <p>
+      You can start this application whenever you are ready. For more info
+      please visit the lin
```
Comment (micahchiang) [inline src/applications/vre/28-1900/wizard/pages/service-member/03-noIDES.jsx:9]: lin should be `link`

File: src/applications/vre/28-1900/wizard/pages/service-member/03-notInterestedInHelp.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ErrorableRadioButtons from '@department-of-veterans-affairs/formation-react/ErrorableRadioButtons';
+import { serviceMemberPathPageNames } from '../pageList';
+
+const notInterestedInHelp = ({ setPageState, state = {} }) => (
+  <div className="feature">
+    <p>
+      You can start this application whenever you are ready. For more info
+      please visit the lin
```
Comment (micahchiang) [inline src/applications/vre/28-1900/wizard/pages/service-member/03-notInterestedInHelp.jsx:9]: lin should be `link`

File: src/applications/vre/28-1900/wizard/pages/veteran/01-noHonorableDischarge.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { veteranPathPageNames } from '../pageList';
+
+const noHonorableDischarge = ({ setPageState, state = {} }) => (
+  <div className="feature">
+    <p>
+      To apply for VR&E benefits, you must have received an other than
```
Comment (micahchiang) [inline src/applications/vre/28-1900/wizard/pages/veteran/01-noHonorableDischarge.jsx:7]: `an other than` should be bolded.

File: src/applications/vre/28-1900/wizard/pages/veteran/03-noActiveDutySeparation.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
Comment (micahchiang) [inline src/applications/vre/28-1900/wizard/pages/veteran/03-noActiveDutySeparation.jsx:1]: This component and the one above have identical content. Is there a way to just use one for both wizard routes?

File: src/applications/vre/28-1900/wizard/pages/veteran/04-notInterestedInEmploymentHelp.jsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ErrorableRadioButtons from '@department-of-veterans-affairs/formation-react/ErrorableRadioButtons';
+import { veteranPathPageNames } from '../pageList';
+
+const notInterestedInEmploymentHelp = ({ setPageState, state = {} }) => (
+  <div className="feature">
+    <p>
+      You can start this application whenever you are ready. For more info
+      please visit the lin
```
Comment (micahchiang) [inline src/applications/vre/28-1900/wizard/pages/veteran/04-notInterestedInEmploymentHelp.jsx:9]: lin should be `link`

**Review by micahchiang — CHANGES_REQUESTED** [review]
Requesting changes to fix some misspelled words, and rename some of the components used to be less generic and more specific to Chapter 31.

File: src/applications/static-pages/static-pages-entry.js [hunk 6]
```diff
@@ -51,6 +51,8 @@ import createBasicFacilityListWidget from './facilities/basicFacilityList';
 import facilityReducer from './facilities/reducers';
 import createOtherFacilityListWidget from './facilities/otherFacilityList';
 
+import createTheWizard from '../vre/28-1900/wizard/app';
```
[the remark this answers is not in the document]
Comment (cohnjesse) [reply src/applications/static-pages/static-pages-entry.js:54]: Agreed, changed this

File: src/applications/static-pages/static-pages-entry.js [hunk 7]
```diff
@@ -161,6 +163,8 @@ createViewTestAndLabResultsPage(
   widgetTypes.VIEW_TEST_AND_LAB_RESULTS_PAGE,
 );
 
+createTheWizard(store, widgetTypes.CHAPTER_31_WIZARD);
```
[the remark this answers is not in the document]
Comment (cohnjesse) [reply src/applications/static-pages/static-pages-entry.js:166]: Agreed, changed this

File: src/applications/vre/28-1900/wizard/app.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+export default function createTheWizard(store, widgetType) {
+  const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
+
+  if (root) {
+    console.log('hey there');
+    import(/* webpackChunkName: "Form686CTA" */
+    './theWizard.jsx').then(module => {
```
[the remark this answers is not in the document]
Comment (cohnjesse) [reply src/applications/vre/28-1900/wizard/app.js:11]: Agreed, changed this

File: src/applications/vre/28-1900/wizard/app.js [hunk 9]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+export default function createTheWizard(store, widgetType) {
+  const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
+
+  if (root) {
+    console.log('hey there');
+    import(/* webpackChunkName: "Form686CTA" */
```
[the remark this answers is not in the document]
Comment (cohnjesse) [reply src/applications/vre/28-1900/wizard/app.js:10]: amazed the linter didn't catch this. Removed it.

File: src/applications/vre/28-1900/wizard/pages/veteran/03-noActiveDutySeparation.jsx (hunk 4 above)
Comment (cohnjesse) [reply src/applications/vre/28-1900/wizard/pages/veteran/03-noActiveDutySeparation.jsx:1]: The content in these may change so until its final I figured we should hold off on refactoring like this.

File: src/applications/vre/28-1900/wizard/pages/veteran/04-notInterestedInEmploymentHelp.jsx (hunk 5 above)
Comment (cohnjesse) [reply src/applications/vre/28-1900/wizard/pages/veteran/04-notInterestedInEmploymentHelp.jsx]: updated

File: src/applications/vre/28-1900/wizard/pages/veteran/01-noHonorableDischarge.jsx (hunk 3 above)
Comment (cohnjesse) [reply src/applications/vre/28-1900/wizard/pages/veteran/01-noHonorableDischarge.jsx]: updated.

File: src/applications/vre/28-1900/wizard/pages/service-member/03-notInterestedInHelp.jsx (hunk 2 above)
Comment (cohnjesse) [reply src/applications/vre/28-1900/wizard/pages/service-member/03-notInterestedInHelp.jsx]: yep, updated this

File: src/applications/vre/28-1900/wizard/pages/service-member/03-noIDES.jsx (hunk 1 above)
Comment (cohnjesse) [reply src/applications/vre/28-1900/wizard/pages/service-member/03-noIDES.jsx]: duh LOL updated this.

File: src/applications/vre/28-1900/wizard/pages/pageList.js [hunk 10]
```diff
@@ -0,0 +1,32 @@
+export const startingPageName = {
+  start: 'start',
+};
+
+export const veteranPathPageNames = {
+  amVeteran: 'amVeteran',
+  amServiceMember: 'amServiceMember',
+  amOther: 'amOther',
```
[the remark this answers is not in the document]
Comment (cohnjesse) [reply src/applications/vre/28-1900/wizard/pages/pageList.js]: yeah you are probably right, updated this.

File: src/applications/vre/28-1900/wizard/app.js [hunk 11]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+export default function createChapter31Wizard(store, widgetType) {
+  const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
+
+  if (root) {
+    import(/* webpackChunkName: "Form686CTA" */
```
Comment (micahchiang) [inline src/applications/vre/28-1900/wizard/app.js:9]: Can you update the value for `webpackChunkName` to Chapter31Wizard

**Review by micahchiang — APPROVED** [review]
lgtm

File: src/applications/vre/28-1900/wizard/app.js (hunk 11 above)
Comment (cohnjesse) [reply src/applications/vre/28-1900/wizard/app.js]: updated

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**